workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -107,83 +168,49 @@ void OnReceived(object sender, DataReceivedEventArgs args)
         /// <summary>
         ///     Runs the redirected process and returns a task which can be waited on.
         /// </summary>
-        /// <param name="workingDirectory">The directory to run the command from.</param>
-        /// <param name="command">The filename to run.</param>
-        /// <param name="arguments">Parameters that will be passed to the command.</param>
         /// <param name="redirectStdout">Redirect standard output to Debug.Log</param>
         /// <param name="redirectStderr">Redirect standard error to Debug.LogError</param>
         /// <returns>A task which would return the exit code and output.</returns>
-        public static async Task<RedirectedProcessResult> RunInAsync(string workingDirectory, string command, string[] arguments, bool redirectStdout = false, bool redirectStderr = false)
+        public async Task<RedirectedProcessResult> RunInAsync(bool redirectStdout = false, bool redirectStderr = false)
```

Comment:
Shouldn't this be RunAsync and the In part of it is folded back into whether or not that option has been set?

Reply:
Kept the name as is - I'll rename it